app/services/hyrax/collections/permissions_service.rb

```diff
@@ -0,0 +1,139 @@
+      # @return [Array<String>] array of group identifiers (typically groupname) for groups who can deposit to this collection
+      def self.group_deposit_grants_for_collection(collection: nil)
+        return [] unless collection
+        # Stubbed to return no grants
+        []
+      end
+
+      # @api public
+      #
+      # Get a list of groups that should be added as group viewers for a collection
+      #
+      # @param collection [Hyrax::Collection] the collection for which permissions are being set
+      # @return [Array<String>] array of group identifiers (typically groupname) for groups who can view this collection
+      def self.group_view_grants_for_collection(collection: nil)
+        return [] unless collection
+        # Stubbed to return no grants
+        []
+      end
+
+      # @api public
+      #
+      # Set the default permissions for a (newly created) collection
+      #
+      # @param collection [Collection] the collection the new permissions will act on
+      # @param creating_user [User] the user that created the collection
+      # @return [Hyrax::PermissionTemplate]
+      def self.create_default(collection:, creating_user:)
+        collection_type = Hyrax::CollectionType.find_by_gid!(collection.collection_type_gid).id
+        PermissionTemplate.create!(source_id: collection.id, source_type: 'collection',
+                                   access_grants_attributes: access_grants_attributes(collection_type: collection_type,
+                                                                                      creating_user: creating_user))
+      end
+
+      # @api private
+      #
+      # Gather the default permissions needed for a new collection
+      #
+      # @param collection_type [CollectionType] the collection type of the new collection
+      # @param creating_user [User] the user that created the collection
+      # @return [Hash] a hash containing permission attributes
+      def self.access_grants_attributes(collection_type:, creating_user:)
+        [
+          { agent_type: 'group', agent_id: admin_group_name, access: Hyrax::PermissionTemplateAccess::MANAGE }
+        ].tap do |attribute_list|
+          # Grant manage access to the creating_user if it exists
+          if creating_user
+            attribute_list << { agent_type: 'user', agent_id: creating_user.user_key, access: Hyrax::PermissionTemplateAccess::MANAGE }
+          end
+        end + managers_of_collection_type(collection_type: collection_type)
+      end
+      private_class_method :access_grants_attributes
+
+      # @api private
+      #
+      # Retrieve the users or groups with manage permissions for a collection type
+      #
+      # @param collection_type [CollectionType] the collection type of the new collection
+      # @return [Hash] a hash containing permission attributes
+      def self.managers_of_collection_type(collection_type:)
```

Comment:
Direct access of the participants table for collection types should not happen in this class.  [Hyrax::CollectionTypes::PremissionsService](https://github.com/samvera/hyrax/blob/collections-sprint/app/services/hyrax/collection_types/permissions_service.rb) handles access to that table.  If you need something that isn't already part of the service, it should be added to the collection types service instead of here.